dev-tools/composer.json

```diff
@@ -1,8 +1,9 @@
 {
     "require": {
-        "php": "^7.0"
+        "php": "^7.1"
```

Comment:
Needed to update because of requirement in `localheinz/composer-normalize`. See https://github.com/localheinz/composer-normalize/pull/30.